interpreters/bas/bas_statement.c

```diff
@@ -1241,7 +1241,7 @@ struct Value *stmt_EDIT(struct Value *value)
 
   return (struct Value *)0;
 #else
-  return Value_new_ERROR(value, NOTAVAILABLE, strerror(ENOSYS));
+  return Value_new_ERROR(value, NOTAVAILABLE);
```

Comment:
Maybe modify `#define NOTAVAILABLE       RUNTIME+22,_("Feature not available")` to `#define NOTAVAILABLE       RUNTIME+22,_("Feature not available %s")` similar to `STRAYSUBEXIT`?
However only `strerror(ENOSYS)` is printed, so does not make sense now.

Reply:
Yes, I think so.